tests/layer_tests/tensorflow_tests/test_tf_Xlogy.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestXlogy(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x' in inputs_info
+        assert 'y' in inputs_info
+        x_shape = inputs_info['x']
+        y_shape = inputs_info['y']
+        inputs_data = {}
+        # generate x in way to have zeros
+        inputs_data['x'] = np.random.randint(-10, 10, x_shape).astype(self.input_type) * \
+                           np.random.randint(0, 2, x_shape).astype(self.input_type)
+        inputs_data['y'] = np.random.randint(-10, 10, y_shape).astype(self.input_type)
```

Comment:
please use float generation: https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.random.html
x from range [-3, 3] and y from range [1, 2] (let us exclude 0 because we have a bug in layer test results comparation that nans are not equal).